web/src/lib/utils/analytics.ts

```diff
@@ -2261,13 +2261,13 @@ export default {
         },
       },
     },
-    quick_action_button: {
+    anytime_action_button: {
       click: {
-        go_to_quick_action_screen: (fileName: string) => {
+        go_to_anytime_action_screen: (fileName: string) => {
           eventRunner.track({
-            legacy_event_category: "quick_action_button",
+            legacy_event_category: "quick_action_button", // Leaving as "quick_action_button" for historical tracking purposes
```

Comment:
Noted as a comment in the code, leaving these values as is for historical data tracking purposes. I'm going to follow up with Aleks to see if this is really an issue.